### DISCUSSION TITLE
Best Octopus Outgoing rate?

### DISCUSSION THREAD

**Discussion by TheSargey** [springfall2008/batpred#1188] [General]
Hi folks,

Just wondered what the general consensus is on which outgoing rate to choose between Outgoing Agile and Outgoing Fixed when using PredBat?  My system, although I've had it installed well over a year, didn't have an export MPAN but I've finally been able to get this arranged and will be applying for an outgoing tariff in the next couple of days.  

I'm assuming that the Outgoing Agile, when managed properly via Predbat should be more effective at generating income than the flat rate?  Mainly as it has more opportunity to export above the 15p limit that you receive on Fixed.  Although the counter argument would be that to export at the highest rates (i.e. between 4pm-8pm) this is most likely the time when one would be using the most energy, so you'd effectively be net flat as I'd probably be importing almost as much as I could possibly export.  I appreciate PredBat can make assumptions around this based on calculated usage based over the past 7,14,21 days - just wondering if anyone has any real world advice on what they prefer?

Many thanks for any advice in advance.

Sarge

**Comment by gcoan** [comment]
Have a look at https://energy-stats.uk/octopus-agile-outgoing-export-eastern-england/ (with the right region for you)

The graphs are not loading for me today, but my recollection is that Agile outgoing is almost always less than then 15p you can get on Agile fixed so the fixed tariff is the best for most people with excess solar.  If you think about it, a sunny day when you are generating loads means the big solar farms are also generating loads and the outgoing rate is lower as a result

**Reply by TheSargey** [reply]
Morning Geoff,

Thanks for the advice (as always) and yes that makes perfect sense. I kind of realised this yesterday when the Agile rates were paying for importing - makes sense to just have a flat rate throughout.

Just hope that Octopus will approve my application without an MCS.

Thanks again for all your help.

Sarge

**Reply by gcoan** [reply]
Morning, don't make a decision based on the negative price periods which are infrequent and unfortunately unpredictable. The long term trend is what's more important and I think it strongly leans to Fixed. If you don't have a lot of excess solar and can choose when to discharge it then outgoing agile might be worth it. 
The other one to look at is Flux. I was on Flux last summer and was very happy with the export income. It's the import rate I didn't like. This year I've decided not to go to Flux, am staying on Agile as the benefits were marginal, but having said that I'm toying with Intelligent Octopus Flux for the better export rate. 

Good luck with getting your export setup

**Reply by TheSargey** [reply]
Ok, I'll have a deeper dive. Thanks for the link, that'll help no end.

All the best,

Sarge

**Reply by gcoan** [reply]
Hi Sarge

You might also want to look at the excellent Octoprice app (can run in a browser or install on your phone/tablet) https://www.octopriceuk.app/

Here for example is the price comparison for Export and Import for the last month for me.  As you can see Fixed export is much better than Agile export,  and Flux import-export is about the same as my current Agile+fixed export

![image](https://github.com/springfall2008/batpred/assets/142018870/66cb4141-a938-4218-b9f2-2aa231519946)
![image](https://github.com/springfall2008/batpred/assets/142018870/b4df987f-74b8-472d-90b8-8c41dcf13bf5)
